javascript/src/main/components/BasicCourseSearch/CourseFilters.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Form} from "react-bootstrap";
```

Comment:
```suggestion
import { Form } from "react-bootstrap";
```

Reply:
Maybe we should keep those fields as a place for folks to do more work on this storybook component in a future commit, along the lines that Steven suggests.